### PR TITLE
Add task in order to delete unconfirmed users (older than 2days)

### PR DESCRIPTION
**Pull request by d0p1s4m4**
Related to #1565 

just add a new rake task in order to delete unconfirmed users

### CODE REVIEW

File: lib/tasks/mastodon.rake [hunk 1]
```diff
@@ -74,6 +74,13 @@ namespace :mastodon do
       end
     end
   end
+  
+  namespace :users do
+    desc 'clear unconfirmed users'
+    task clear: :environment do
+      User.where('confirmed_at is NULL AND confirmation_sent_at <= ?', 7.days.ago).find_each(&:destroy)
```
Comment (Gargron) [inline lib/tasks/mastodon.rake]: Hmm maybe 24 hours or 2 days? Highly unlikely that someone would press "confirm" in an e-mail after waiting for so long.

Comment (Angristan) [inline lib/tasks/mastodon.rake]: Yes, 2 days seems good.

Comment (Golgotha29) [inline lib/tasks/mastodon.rake]: So, add a line on the first mail or on the web page : "You have two days for validate your account..."

**Review by Gargron — APPROVED** [review]

**Comment by Fastidious** [comment]
Can you update the documentation on this one? Some administration document or similar, perhaps?

**Comment by symac** [comment]
Hello,
will there be any way for a remote service to know that a user account has been deleted ? On http://mastory.social/ I am regularly looking at user pages on their instances to update the information I have and if I get a 404, at the moment, I consider it might be a temporary issue with the server and try again later. If the user has been deleted I would be interested to know it and be able to remove it from my database.

From what I can see in the diff it is just deleting the user from DB, is there any plan to change/improve that ?

**Comment by Wonderfall** [comment]
@Fastidious `docker-compose run --rm mastodon-web rails mastodon:users:clear`

**Comment by Gargron** [comment]
@symac You shouldn't be able to discover unconfirmed users, since they can't login (therefore, can't post)

Suspended accounts return 410.

**Comment by valentin2105** [comment]
Can you release v1.1.2 with this merge ? 

Thanks a lot.

**Pull request merged**